extensions/ql-vscode/src/run-remote-query.ts

```diff
@@ -413,3 +434,23 @@ export async function attemptRerun(
     void showAndLogErrorMessage(error);
   }
 }
+
+/**
+ * Updates the default suite of the query pack. This is used to ensure
+ * only the specified query is run.
+ *
+ * @param queryPackDir The directory containing the query pack
+ * @param packRelativePath The relative path to the query pack from the root of the query pack
```

Comment:
The wording looks a bit confusing here 😅  Should this be... ?
```suggestion
 * @param packRelativePath The relative path to the query from the root of the query pack
```